asf_public_discourse_home_decarbonisation/pipeline/bert_topic_analysis/title_topic_analysis.py

```diff
@@ -0,0 +1,159 @@
+"""
+Script for identifying topics of conversation using forum thread titles as input.
+
+The pipeline:
+- Gets forum data;
+- Applies topic analysis to unique titles to identify topics of conversation
+- Outputs are saved to S3 including information about topics the titles data
+
+To run this script:
+python asf_public_discourse_home_decarbonisation/pipeline/bert_topic_analysis/title_topic_analysis.py --source SOURCE --start_date START_DATE --end_date END_DATE --reduce_outliers_to_zero REDUCE_OUTLIERS_TO_ZERO --min_topic_size MIN_TOPIC_SIZE
+
+Example usage:
+python asf_public_discourse_home_decarbonisation/pipeline/bert_topic_analysis/title_topic_analysis.py --source "mse" --end_date "2024-05-22" --min_topic_size 300
+
```

Comment:
Potentially putting an example for buildhub
```suggestion
For buildhub, we will want to pick a lower minimum topic size as it's not as big of a forum as money saving expert:
python title_topic_analysis.py --source "buildhub" --end_date "2024-05-22" --min_topic_size 30
```

Reply:
I tried to only add examples of things we have actually ran. So I only didn't add an example for Buildhub because we haven't used this for Buildhub. Of course we can, and hopefully people understand that's possible since we explain what source is and provide Buildhub and MSE as sources.